stdlib/asm/crypto/fri/helper.masm

```diff
@@ -109,7 +109,6 @@ export.load_fri_layer_commitments
         exec.random_coin::reseed
         # => [num_layers, ptr_layer + 1, y, y, ...]
 
-        push.0.0.0.0
         push.0.0.0.0
         exec.random_coin::get_rate_1
```

Comment:
`get_rate_1` now does the `padw` [ref](https://github.com/0xPolygonMiden/miden-vm/pull/965/files#diff-f71876b54afaf7103caaa9741a02c8f9d7c5c126e59c12a6f54575f6ec5bc0bbR41)